examples/user_defined_type.sol

```diff
@@ -0,0 +1,10 @@
+
```

Comment:
This file is not formatted correctedly unless the first blank line is removed

Reply:
Hmm yes. There might be rare cases where the formatter can't handle the file :shrug: Thanks for pointing out.